scripts/partition_manager/partition_manager.rst

```diff
@@ -327,6 +327,7 @@ RAM partition configuration
           size: 0xa000
 
 All occurrences of a partition name can be replaced with a dict with the key ``one_of``, which is resolved to the first existing partition in the ``one_of`` value.
+The value of the ``one_of`` key must be a list of placeholder or image partitions, and it cannot be a span.
```

Comment:
in addition to this, would it be possible to detect a malformed pm.yml here:
https://github.com/nrfconnect/sdk-nrf/blob/master/scripts/partition_manager.py

and fail gracefully with a proper error message to the user ?

Reply:
Sure, I think that would be a very useful feature to add.

I think adding it here is out of scope of this Jira task and PR, but I would be happy to open a separate Jira task for implementing this functionality down the road.

Reply:
Opened NCSDK-7398 to address this feature implementation.